resources/authgear/templates/en/translation.json

```diff
@@ -270,6 +273,10 @@
   "password-policy-password-strength-meter-3": "Fair",
   "password-policy-password-strength-meter-4": "Very unguessable",
   "password-policy-password-strength-meter-5": "Extremely unguessable",
+  "v2-password-policy-password-strength-meter-1": "Very Weak",
+  "v2-password-policy-password-strength-meter-2": "Weak",
+  "v2-password-policy-password-strength-meter-3": "Medium",
+  "v2-password-policy-password-strength-meter-4": "Strong",
```

Comment:
- As mentioned in slack, we need 5 level.
- Please put all v2-.. translation strings at the bottom.